### PR TITLE
Task general

### PR DESCRIPTION
**Pull request by Yas-Shen**
Nav bar, Header, About, portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+				<a href="#contact">Contact</a>
+				<a href="javascript:void(0);" class="icon" onclick="myFunction()">
+					<i class="fa fa-bars"></i>
+				</a>
+		</div>
+	</nav>
+	<div class="main-box">
+		<header>Yasmin Shenair - Web Developer</header>
+	<p>Hello, this is my portfolio, stay tuned for projects! Hello, this is my portfolio, stay tuned for projects! Hello, this is my portfolio, stay tuned for projects! Hello, this is my portfolio, stay tuned for projects!</p>
+</div>
```
Comment (Joperes-ym) [inline index.html:34]: The main-box and project-section are not centered.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		<h2>Project</h2>
+		<a href="#" class="code-image" title="code-image"><img src="coding.jpg" alt="code image"></a>
+		<p>Hello, this is my portfolio, stay tuned for projects!</p>
+		<div class="buttons"><button class="live">View Live</button>
+		<button class="code">View Code</button>
+	</div>
+	</div>
+	<div class="project-item">
+		<h2>Project</h2>
+		<a href="#" class="code-image" title="code-image"><img src="coding.jpg" alt="code image"></a>
```
Comment (Joperes-ym) [inline index.html:47]: Images on project are not centered

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+	align-items: center;
+}
+
+.code-image img {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	width: 60%;
+	height: 60%;
+	margin-left: 45px;
```
Comment (Joperes-ym) [inline styles.css:121]: Images on project not centered - would be because of this margin?

File: index.html (hunk 2 above)
Comment (Joperes-ym) [reply index.html:47]: desktop version - margin too big on the left
mobile version - margin too big on the right side

File: index.html (hunk 1 above)
Comment (Yas-Shen) [reply index.html:34]: Think I have sorted it out.

File: styles.css (hunk 3 above)
Comment (nesclark) [inline styles.css:121]: The image inside the project containers are not centered. Could this be fixed by using position: relative?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+	  <link rel="stylesheet" type="text/css" href="styles.css">
+	  <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet" type="text/css">
+	  <link rel="stylesheet" type="text/css" href="normalize.css">
+</head>
+<body>	
+	<nav>
+		<div class="initials">
+			<h3 class="name">Yasmin Shenair</h3>
+			<a href="#" class="icon" title="initials">
+				<img src="ys.png"alt="initials" />
```
Comment (nesclark) [inline index.html:16]: you might want a space between 'src' and 'alt' attributes 

e.g. img src="ys.png" alt="initials"

File: index.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+			</a>
+			<a href="#" class="icon" title="icon">
+				<img src="hamburger.png"alt="hamburger-menu" />
```
Comment (nesclark) [inline index.html:19]: same again here - needs a space between 'src' and 'alt' attributes

File: styles.css (hunk 3 above)
Comment (Yas-Shen) [reply styles.css:121]: Turns out it's already in my code, for some reason it didn't update on here.
This is what I found when I checked it out... .code-image img {
	display: flex;
	justify-content: center;
	align-items: center;
	position: relative;
	left: 10px;
	width: 80%;
	height: 80%;
	margin: 0 0 0 7%;
}

File: index.html (hunk 4 above)
Comment (Yas-Shen) [reply index.html:16]: Done.

File: index.html (hunk 5 above)
Comment (Yas-Shen) [reply index.html:19]: And done lol..

**Pull request merged**